admin_guide/overcommit.adoc

```diff
@@ -337,16 +337,26 @@ limit. Each container may start using swap. Eventually, if there is not enough
 swap space, processes in the pods can be terminated (due to the system being
 oversubscribed).
 
-There are options that can help you avoid having to swap, such as moving pods to
-nodes with free resources, adding physical memory, reducing `*vm.swappiness*`,
-the use of huge pages, or disabling `*vm.overcommit*`.
+It is recommended to disable swap by default on your nodes in order to preserve
+quality of service guarantees.
 
-[WARNING]
-====
-Swap can also be disabled, but that is not recommended. Disable swap memory on
-each node by running:
+To disable swap:
 
 ----
 $ swapoff -a
 ----
+
+[WARNING]
 ====
+Failure to disable swap will result in nodes not recognizing that they are experiencing
```

Comment:
@jeremyeder - here as well, note the toggle in the recommendation to disable swap.

/cc @sdodson